src/main/java/org/ideoholic/curium/model/student/service/StudentService.java

```diff
@@ -842,79 +841,77 @@ public StudentDetailsResponseDto viewDetailsOfStudent(String studentId) {
 			//String totalFees = new feesDetailsDAO().feesTotal(id, currentYear.getCurrentacademicyear());
 			//String dueAmount = new feesDetailsDAO().dueAmount(id, currentYear.getCurrentacademicyear());
 			if (parents == null) {
-				response.setSuccess(false);
+				result.setSuccess(false);
 			} else {
-				response.setStudent(parents.getStudent());
+				result.setStudent(parents.getStudent());
 				String classStudying = parents.getStudent().getClassstudying();
 				if (!classStudying.equalsIgnoreCase("")) {
 					String[] classParts = classStudying.split("--");
-					response.setClassStudying(classStudying);
-					response.setClassParts(classParts[0]);
-					response.setSecStudying("");
+					result.setClassStudying(classStudying);
+					result.setClassParts(classParts[0]);
```

Comment:
It should be: `result.setClassStudying(classParts[0]);`
The same should be set in adapter as: `httpSession.setAttribute("classstudying", result.getClassStudying());`